rq/worker.py

```diff
@@ -483,7 +481,7 @@ def work(self, burst=False, logging_level="INFO"):
                         self.log.info('Stopping on request')
                         break
 
-                    timeout = None if burst else max(1, self.default_worker_ttl - 60)
+                    timeout = None if burst else max(1, self.default_worker_ttl - 15)
```

Comment:
This is perhaps an ignorant question, since I'm new to this codebase, but why hardcode this value at all?   Would there be a reason not to do something like:

```timeout = None if burst else max(1, int(self.default_worker_ttl / 2) - 1)```